src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -154,7 +156,17 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         if ('doctrine_orm' === $config['db_driver']) {
-            $this->registerDoctrineMapping($config);
+            if (isset($bundles['SonataDoctrineBundle'])) {
+                $this->registerSonataDoctrineMapping($config);
+            } else {
+                // NEXT MAJOR: Throw error when not registering SonataDoctrineBundle
+                @trigger_error(
```

Comment:
Could we make a safer check? Like `isset($bundles['SonataEasyExtendsBundle'])` instead of rely on the absence of "SonataDoctrineBundle".

Reply:
I can add it, but the previous code did not rely on the presence on the easy extends

Reply:
I understand. I'm just trying to point that, at first glance, telling the user
>Registering `SonataEasyExtendsBundle` bundle is now deprecated

in case "SonataDoctrineBundle" is not registered, seems confusing.

Reply:
Changing the UPGRADE note should be enough, this way we don't alter previous behavior. To prevent BC breaks :)